### PR TITLE
Update files with PEP8 standards in folder dojo/tools #002

### PR DESCRIPTION
**Pull request by ajtortolero**
Update files with PEP8 standars in folders dojo/tools/

Folder affected
dojo/tools/dockle
dojo/tools/drheader
dojo/tools/dsop
dojo/tools/eslint
dojo/tools/fortify
dojo/tools/generic
dojo/tools/ggshield
dojo/tools/github_vulnerability
dojo/tools/gitlab_api_fuzzing
dojo/tools/gitlab_container_scan
dojo/tools/gitlab_dast
dojo/tools/gitlab_dep_scan
dojo/tools/gitlab_sast
dojo/tools/gitlab_secret_detection_report

### CODE REVIEW

**Comment by Maffooch** [comment]
Failing unit tests are unrelated. They refuse to fully run for whatever reason. Will likely resolve itself on the next commit

File: dojo/tools/gitlab_sast/parser.py [hunk 1]
```diff
@@ -26,35 +24,33 @@ def get_findings(self, json_output, test):
...
         test.findings = self.get_items(tree)
         return [test]
 
     def parse_json(self, json_output):
         data = json_output.read()
         try:
-            tree = json.loads(str(data, 'utf-8'))
-        except:
+            tree = json.loads(str(data, "utf-8"))
+        except BaseException:
```
[the remark this answers is not in the document]
Comment (ajtortolero) [reply dojo/tools/gitlab_sast/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8301/commits/965230ac2695ecd73cde22c96ff4b2aa25b92025

File: dojo/tools/gitlab_dep_scan/parser.py [hunk 2]
```diff
@@ -26,92 +25,108 @@ def parse_json(self, json_output):
         try:
             data = json_output.read()
             try:
-                tree = json.loads(str(data, 'utf-8'))
-            except:
+                tree = json.loads(str(data, "utf-8"))
+            except BaseException:
                 tree = json.loads(data)
-        except:
+        except BaseException:
```
[the remark this answers is not in the document]
Comment (ajtortolero) [reply dojo/tools/gitlab_dep_scan/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8301/commits/965230ac2695ecd73cde22c96ff4b2aa25b92025

File: dojo/tools/gitlab_dep_scan/parser.py [hunk 3]
```diff
@@ -26,92 +25,108 @@ def parse_json(self, json_output):
         try:
             data = json_output.read()
             try:
-                tree = json.loads(str(data, 'utf-8'))
-            except:
+                tree = json.loads(str(data, "utf-8"))
+            except BaseException:
```
[the remark this answers is not in the document]
Comment (ajtortolero) [reply dojo/tools/gitlab_dep_scan/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8301/commits/965230ac2695ecd73cde22c96ff4b2aa25b92025

File: dojo/tools/eslint/parser.py [hunk 4]
```diff
@@ -25,42 +24,45 @@ def _convert_eslint_severity_to_dojo_severity(self, eslint_severity):
     def get_findings(self, filename, test):
         tree = filename.read()
         try:
-            data = json.loads(str(tree, 'utf-8'))
-        except:
+            data = json.loads(str(tree, "utf-8"))
+        except BaseException:
```
[the remark this answers is not in the document]
Comment (ajtortolero) [reply dojo/tools/eslint/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8301/commits/965230ac2695ecd73cde22c96ff4b2aa25b92025

**Review by Maffooch — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Review by blakeaowens — APPROVED** [review]

**Review by alles-klar — APPROVED** [review]

**Pull request merged**